harness/attributes/common.yml

```diff
@@ -3,14 +3,15 @@ attributes.default:
     binary: app
     src_path: .
     module_path: = @('app.src_path')
-    port: 80
     health_port: 80
     services: []
+    default_port:
+      port: 80
```

Comment:
may read better if this is `number`?

Reply:
fits more 1 to 1 with the k8s configuration as `port`

Reply:
`{{ .Values.default_port.number }}` reads better than `{{ .Values.default_port.port }}` imo